VT-LPR/README.md

```diff
@@ -0,0 +1,109 @@
+
+## Requirements
+* Python>=3.6
+* Streamlit==1.0.0
+* PyQt5==5.15.4
+* PyQt5Designer==5.14.1
+* OpenCV==4.5.1.48
+* pillow
+* matplotlib
+* pandas
+* numpy
+* tensorflow
+* tflite
+
+Note: These packages can be installed through Pip package installer. This code is tested with these versions.
+
+## Challenges
+Image based vehicle type and license plate recognition include many challenges such as:
+* Intra-class variations
+* Background clutter
+* Non-uniform position
+* Multiple license plates
+* Non-uniform fonts
+* Partial occlusion
+* Damaged license plates
+* Lack of license plates
+
+<p align="center">
+  <img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/problems.jpg" title="hover text">
+  <img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/variations.jpg" title="hover text">
+</p>
+
+## Methodology
+### Dataset
+We divide the Vehicle type and license plate recognition task into three steps: Vehicle type recognition, License plate localization and License plate character recognition. So we prepared three dataset named: _preprocessed dataset for Vehicle type recognition, lp-detect dataset for license plate location and lp-read dataset for license plate character recognition._
+As we're training separate model for each task, all the vehicle images are annotated accordingly as shown below.
+
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/label.jpg" title="hover text">
+
+### Framework
+
+The proposed framework is aim to automatically collect toll amount by end-to-end image-based vehicle type and license plate recognition. Input to the model is an image and the outputs are recognized vehicle type and license plate character. When an image is fed to the first network called VT-Net, it recognize the vehicle type and crop the detection results. The cropped image is then fed to the secong network called LP-Net, which localize and crop the license plate. Finally, the last model called CR-Net is applied on the cropped license plate image to recognize the characters in the license plate.
+
+<p align="center">
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/flow.jpg" width ="400" title="hover text">
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/chart.jpg" width ="450" title="hover text">
+</p>
+
+### Results
+<p align="center">
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/type_res.jpg" width ="400" title="hover text">
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/lp_res.jpg" width ="320" title="hover text">
+  <img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/lpr_res.jpg" width ="450" title="hover text">
+</p>
+
+## Test
+
+1. Download and extract trained model weights from [Google Drive](https://drive.google.com/file/d/1nGe7kQaKwKzzrvPYSd41ScuSuj2D33-C/view?usp=sharing). 
+
+``` #2x
+https://drive.google.com/file/d/1nGe7kQaKwKzzrvPYSd41ScuSuj2D33-C/view?usp=sharing
+```
```

Comment:
This segment is no longer necessary with DagsHub, since you uploaded the model weights, so you can remove this and start from the next section